gulpfile.js

```diff
@@ -63,128 +63,134 @@ var unminifiedRuntimeTarget = 'dist/amp.js';
+declareExtension('amp-reach-player', '0.1', {hasCss: false});
+declareExtension('amp-reddit', '0.1', {hasCss: false});
+declareExtension('amp-riddle-quiz', '0.1', {hasCss: false});
+declareExtension('amp-share-tracking', '0.1', {hasCss: false});
+declareExtension('amp-sidebar', '0.1', {hasCss: true});
+declareExtension('amp-soundcloud', '0.1', {hasCss: false});
+declareExtension('amp-springboard-player', '0.1', {hasCss: false});
+declareExtension('amp-sticky-ad', '1.0', {hasCss: true});
+declareExtension('amp-story', '0.1', {hasCss: true});
+declareExtension('amp-selector', '0.1', {hasCss: true});
+declareExtension('amp-web-push', '0.1', {hasCss: true});
+declareExtension('amp-wistia-player', '0.1', {hasCss: false});
+declareExtension('amp-position-observer', '0.1', {hasCss: false});
+declareExtension('amp-date-picker', '0.1', {hasCss: true});
+declareExtension('amp-image-viewer', '0.1', {hasCss: true});
 
 /**
  * @deprecated `amp-slides` is deprecated and will be deleted before 1.0.
  * Please see {@link AmpCarousel} with `type=slides` attribute instead.
  */
-declareExtension('amp-slides', '0.1', false);
-declareExtension('amp-social-share', '0.1', true);
-declareExtension('amp-timeago', '0.1', false);
-declareExtension('amp-twitter', '0.1', false);
-declareExtension('amp-user-notification', '0.1', true);
-declareExtension('amp-vimeo', '0.1', false);
-declareExtension('amp-vine', '0.1', false);
-declareExtension('amp-viz-vega', '0.1', true);
-declareExtension('amp-google-vrview-image', '0.1', false);
+declareExtension('amp-slides', '0.1', {hasCss: false});
+declareExtension('amp-social-share', '0.1', {hasCss: true});
+declareExtension('amp-timeago', '0.1', {hasCss: false});
+declareExtension('amp-twitter', '0.1', {hasCss: false});
+declareExtension('amp-user-notification', '0.1', {hasCss: true});
+declareExtension('amp-vimeo', '0.1', {hasCss: false});
+declareExtension('amp-vine', '0.1', {hasCss: false});
+declareExtension('amp-viz-vega', '0.1', {hasCss: true});
+declareExtension('amp-google-vrview-image', '0.1', {hasCss: false});
 declareExtension('amp-viewer-integration', '0.1', {
   // The viewer integration code needs to run asap, so that viewers
   // can influence document state asap. Otherwise the document may take
   // a long time to learn that it should start process other extensions
   // faster.
   loadPriority: 'high',
 });
-declareExtension('amp-video', '0.1', false);
-declareExtension('amp-vk', '0.1', false);
-declareExtension('amp-youtube', '0.1', false);
+declareExtension('amp-video', '0.1', {hasCss: false});
+declareExtension('amp-vk', '0.1', {hasCss: false});
+declareExtension('amp-youtube', '0.1', {hasCss: false});
 declareExtensionVersionAlias(
     'amp-sticky-ad', '0.1', /* lastestVersion */ '1.0', /* hasCss */ true);
+
+
+/**
+ * @typedef {{
+ *   name: ?string,
+ *   version: ?string,
+ *   hasCss: ?boolean,
+ *   loadPriority: ?string
+ * }}
+ */
+const ExtensionOption;
```

Comment:
Looks like a leftover const definition. Mind removing?

Reply:
this is required for jscompiler typechecking (it needs a reference to an identifier)